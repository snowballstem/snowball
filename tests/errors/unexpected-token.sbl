externals (stem)
integers (i)
groupings (g)
define g +
define stem as (
  $
  next
  $(i 0)
  $i = hop 2
  $(i = )
  backwardmode (next)
)
next
