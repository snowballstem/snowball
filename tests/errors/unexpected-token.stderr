errors/unexpected-token.sbl:4: unexpected + in grouping definition after name
errors/unexpected-token.sbl:7: unexpected next in integer test expression after $
errors/unexpected-token.sbl:8: unexpected number 0 in integer test expression after name
errors/unexpected-token.sbl:9: unexpected hop in integer expression after =
errors/unexpected-token.sbl:10: Expected relational operator (did you mean '=='?)
errors/unexpected-token.sbl:10: unexpected ) in integer expression after =
errors/unexpected-token.sbl:11: unexpected backwardmode after )
errors/unexpected-token.sbl:13: unexpected next after )
